dbt/macros/res_report_summary.sql

```diff
@@ -0,0 +1,53 @@
+-- Macros that summarize FMVs and sales by different groupings specifically for
+-- reporting.res_report_summary.
+{% macro res_report_summarize_values(vals, geo_type, prop_group) %}
+    select
+        triad,
+        '{{ geo_type }}' as geography_type,
+        {% if prop_group %} property_group,
+        {% elif not prop_group %} 'ALL REGRESSION' as property_group,
+        {% endif %}
+        assessment_stage,
+        {% if geo_type == "Town" %} township_code as geography_id,
+        {% elif geo_type == "TownNBHD" %} townnbhd as geography_id,
+        {% endif %}
+        year,
+        approx_percentile(total, 0.5) as fmv_median,
+        count(*) as pin_n,
+        approx_percentile(total_land_sf, 0.5) as land_sf_median,
+        approx_percentile(total_bldg_sf, 0.5) as bldg_sf_median,
+        approx_percentile(yrblt, 0.5) as yrblt_median
+    from {{ vals }}
+    group by
+        assessment_stage,
+        triad,
+        {% if geo_type == "Town" %} township_code,
+        {% elif geo_type == "TownNBHD" %} townnbhd,
+        {% endif %}
+        {% if prop_group %} year, property_group
+        {% elif not prop_group %} year
+        {% endif %}
```

Comment:
**[Nitpick, non-blocking]** A slight simplification, if it appeals to you:

```suggestion
        year
        {%- if prop_group %}, property_group{%endif %}
```

Reply:
I had written this oddly in order to try to make it explicit that that expect values should only be True or False, but i don't think it adds much.